api/envoy/extensions/common/dynamic_forward_proxy/v3/dns_cache.proto

```diff
@@ -83,4 +90,8 @@ message DnsCacheConfig {
   // this is used as the cache's DNS refresh rate when DNS requests are failing. If this setting is
   // not specified, the failure refresh rate defaults to the dns_refresh_rate.
   config.cluster.v3.Cluster.RefreshRate dns_failure_refresh_rate = 6;
+
+  // The config of circuit breakers for resolver. It provides a configurable threshold.
+  // If this value is not set, the cluster circuit breakers will be used.
```

Comment:
I don't think this is right. We add a runtime feature flag so that people can migrate, so I think if this is not specified and the feature flag is true we should use the default settings for the DNS cache circuit breaker and ignore the cluster settings.